packages/db/src/search/CollectionSearch.ts

```diff
@@ -0,0 +1,75 @@
+import { Knex } from 'knex';
+import Logger from '@cumulus/logger';
+import { CollectionRecord } from '@cumulus/types/api/collections';
```

Comment:
Ah, ok. That makes it a little confusing but I see why you can't change it.

Reply:
After sitting on this a bit, I think what would be helpful here is a comment somewhere saying that there is no `ApiCollectionRecord` because `CollectionRecord` is the type from the API, and has all the same fields (also 'that are needed', if it's the case that there are other fields but not needed for querying).